tests/__tests__/ts-errors.spec.ts

```diff
@@ -9,9 +9,20 @@ describe('hello_world', () => {
     const stderr = result.stderr.toString();
 
     expect(result.status).toBe(1);
-    expect(stderr).toContain('Hello.ts:13:11');
+    expect(stderr).toContain('Hello.ts:18:11');
```

Comment:
Good catch - the file has changed though. I wanted to be completely sure I didn't break anything, so I added a typescript interface to simple/Hello.ts - I wanted to put something in there that'd fill up a few lines, but I knew would be stripped by tsc